solidity/random-beacon/README.adoc

```diff
@@ -101,39 +100,34 @@ expiration is performed in the relay request transaction.
 
 === Relay Request and Relay Entry
 
-Anyone can request a new relay entry (random number) by calling
+Authorized addresses can request a new relay entry (random number) by calling
 `RandomBeacon.requestRelayEntry(IRandomBeaconConsumer callbackContract)`
-function and providing an optional callback parameter. The requester needs to
-approve enough tokens for a fee, as described in
-<<fees-and-rewards,Fees and Rewards>> section.
+function and providing an optional callback parameter.
 
 In `requestRelayEntry` transaction, groups that reached their maximum lifetime
 are getting expired and one of the remaining active groups is tasked with
 producing a new relay entry. The off-chain clients are expected to monitor the
 `RelayEntryRequested` event. If a client is a part of a picked group they should
 start the off-chain protocol to sign the previous relay entry producing a new one.
 
-Off-chain clients are expected to follow the order when submitting relay entry
-to minimize and distribute costs evenly, as described in
-<<fees-and-rewards,Fees and Rewards>> section but no ordering is enforced
-on-chain. New relay entry should be submitted using 
-`RandomBeacon.submitRelayEntry(bytes calldata entry)`
-function.
+Off-chain clients are expected to follow the submission order when submitting
+relay entry to avoid front-running and minimize the cost but no ordering is
+enforced on-chain. New relay entry should be submitted using 
```

Comment:
Could we link 'submission order' to the Operator-Only section in Transaction Incentives?

Reply:
1fadd0bedacc1e8332f6ef3c24e1f6db4acc3322